initfiles/componentfiles/thor/start_backupnode.in

```diff
@@ -71,7 +71,7 @@ if [ ! -z ${THORPRIMARY} ]; then
 else
     groupName=${THORNAME}
 fi
-daliadmin $DALISERVER dfsgroup ${groupName} > $INSTANCE_DIR/slaves
+daliadmin $DALISERVER dfsgroup ${groupName} $INSTANCE_DIR/backupnode.slaves
```

Comment:
K gotcha, Thought this was for 5.4 for some reason.  Didn't realize it was for 5.2.4.  Everything looks good for a merge.